### ISSUE TITLE
What is `local_ip_address`

### ISSUE THREAD

**Issue by ruslo** [hunter-packages/hunter-bot#10]
We have `public_ip_address` already, why we need local? Is it mean to be `127.0.0.1`?

**Comment by Cyberunner23** [comment]
It's the same ip address a regular webserver would listen to, for the test server the one that works is 0.0.0.0

**Comment by ruslo** [comment]
Can we always have `0.0.0.0`?

**Comment by Cyberunner23** [comment]
it depends on the server you put the bot on, sometimes its configed to use 0.0.0.0 sometimes 127.0.0.1

**Comment by ruslo** [comment]
Yes, this is kind of what I mean. Can we decide what to use between those two without involving user?

**Comment by ruslo** [comment]
Probably if I rephrase the question it will be more helpful: "what should I use for my own server now?"

**Comment by ruslo** [comment]
By the way do we need to use ip in `public_ip_addres`? Can it be hostname? E.g. `bot.hunter.sh`?

**Comment by Cyberunner23** [comment]
> By the way do we need to use ip in public_ip_addres? Can it be hostname? E.g. bot.hunter.sh?

We should test it but I think it will work

**Comment by Cyberunner23** [comment]
> "what should I use for my own server now?"

Probably 127.0.0.1? For it to work on the test server I tried with common ip adresses and 0.0.0.0 worked, you might have to experiment as well
